### ISSUE TITLE
Error FS0039: The namespace 'Tools' is not defined.

### ISSUE THREAD

**Issue by gkonterman** [fsharp/FAKE#1992]
### Description

I have a brand new laptop so with a clean install but my build fails with `Error FS0039: The namespace 'Tools' is not defined.`  I am not exactly sure where the problem is (or which).
The weird thing is that is actually works on my older laptop. 

It is certain that `FAKE.Tools.Git` is the culprit here. Is there something changed?

### Repro steps

`build.fsx`
``` 
#r "paket: groupref netcorebuild //"
#load "./.fake/build.fsx/intellisense.fsx"

open Fake.Tools.Git
```

Invoke
`dotnet fake run build.fsx`

### Expected behavior

Successful build

### Actual behavior

`Error FS0039: The namespace 'Tools' is not defined.`

### Known workarounds

Not using `FAKE.Tools.Git`

### Related information 

`winver / ver`
```
Windows 10
Microsoft Windows [Version 10.0.17134.81]
```

`dotnet --version`
```
2.1.300
```

`git --version`
```
git version 2.17.1.windows.2
```

**Comment by gkonterman** [comment]
Repro [fake.zip](https://github.com/fsharp/FAKE/files/2091582/fake.zip)

Run `bootstrap.cmd`

**Comment by kblohm** [comment]
I did not run the `https://dot.net/v1/dotnet-install.ps1` stuff as i did not want to DL dotnet again. The rest works just fine for me.

**Comment by matthid** [comment]
@gkonterman 

like @kblohm  I see:
```
PS E:\Projects\FAKE\temp\fake> .\bootstrap.cmd
== Install .NET Core SDK
== Restore tools
== Restore Fake
== Compile build script
E:\Projects\FAKE\temp\fake\build.fsx (4,19)-(4,19): Warning FS0988: Main module of program is empty: nothing will happen when it is run
```

Which is basically what I would expect.

**Comment by gkonterman** [comment]
@matthid @kblohm Yes its really weird, on other PCs it works just fine. I just got a new ultra fresh Dell XPS 15 9570, installed Git, VS2017 and cloned the repository. My hope and expection was that I could just run `./build` but I got this error.

I have no clue how to fix this :( To me it seems that there is something not right on this new box, but what? I've check all F# options in the VS installer.
